TheMapCloudWebServices/plugin.cfg

```diff
@@ -0,0 +1,5 @@
+[plugin_configuration]
+name = theMapCloud
+title = theMapCloud Web Services
+help_url = https://api.themapcloud.com
```

Comment:
I'll leave this as is for now, probably want to change per deployment to our QGIS repo